app/main/controllers/template/RTMediaNav.php

```diff
@@ -284,9 +284,7 @@ public function sub_nav() {
 
 		$albums = '';
 		// condition to keep "Album" tab active.
-		if ( array_key_exists( 'media_type', $rtmedia_query->query ) && isset( $rtmedia_query->query['media_type'] ) && ( 'album' === $rtmedia_query->query['media_type'] ) ) {
-			$albums = 'current selected';
-		} elseif ( array_key_exists( 'media_type', $rtmedia_query->action_query ) && isset( $rtmedia_query->action_query->media_type ) && ( 'album' === $rtmedia_query->action_query->media_type ) ) {
+		if ( ( array_key_exists( 'media_type', $rtmedia_query->query ) && isset( $rtmedia_query->query['media_type'] ) && ( 'album' === $rtmedia_query->query['media_type'] ) ) || ( array_key_exists( 'media_type', $rtmedia_query->action_query ) && isset( $rtmedia_query->action_query->media_type ) && ( 'album' === $rtmedia_query->action_query->media_type ) ) ) {
```

Comment:
Could we split into multiple lines for better readability?

Reply:
okay